akka-docs/rst/scala/code/docs/dispatcher/DispatcherDocSpec.scala

```diff
@@ -78,12 +78,7 @@ object DispatcherDocSpec {
       executor = "thread-pool-executor"
       # Configuration for the thread pool
       thread-pool-executor {
-        # minimum number of threads to cap factor-based core number to
-        core-pool-size-min = 2
-        # No of core threads ... ceil(available processors * factor)
-        core-pool-size-factor = 2.0
-        # maximum number of threads to cap factor-based number to
-        core-pool-size-max = 10
+        fixed-pool-size = 16
       }
```

Comment:
Something like this is already defined in `//#fixed-pool-size-dispatcher-config`.
Revert this change.